Cargo.toml

```diff
@@ -9,9 +9,17 @@ authors = [
 license = "MIT"
 description = "Raw unsafe system binding for openvr"
 repository = "https://github.com/rust-openvr/rust-openvr-sys"
-
+exclude = [
+    "openvr/samples/*",
+    "openvr/bin/*",
+    "openvr/lib/*",
```

Comment:
The binaries in the upstream repository are excluded as of 50e98d5 because the changes include building the OpenVR client code from source and statically linking it. This should make redistributing binaries which use this library much easier, as there's no need to patch runpaths or muck about with dynamic library search paths.